.tests/openvpnddwrt-logs/parser.assert

```diff
@@ -0,0 +1,187 @@
+len(results) == 3
+len(results["s00-raw"]["crowdsecurity/syslog-logs"]) == 12
+results["s00-raw"]["crowdsecurity/syslog-logs"][0].Success == true
+results["s00-raw"]["crowdsecurity/syslog-logs"][0].Evt.Parsed["logsource"] == "syslog"
+results["s00-raw"]["crowdsecurity/syslog-logs"][0].Evt.Parsed["message"] == "daemon.err openvpn[1422]: 5.181.98.144:32425 TLS Error: TLS handshake failed"
```

Comment:
Here the example of raw syslog not getting it correctly.

The message should only be `5.181.98.144:32425 TLS Error: TLS handshake failed`

Reply:
Sorry I do not understand.
the machine name (or IP) is not used to trigger an event. I only want to trig an event on  "TLS Error : TLS handshake failed" pattern or "TLS Error : TLS key negotiation failed to occur within 60 seconds" pattern in syslog message.
It's the reason why I use such pattern in parser yaml file : 
pattern_syntax:
  OPENVPNDDWRT_BADKEY: '^%{DATA:program}: %{IP:ip}:%{POSINT:port} TLS Error: TLS key %{DATA:error}$'
  OPENVPNDDWRT_BADHS: '^%{DATA:program}: %{IP:ip}:%{POSINT:port} TLS Error: TLS handshake failed%{DATA:error}$'

Here are some examples of syslog messages as recorded on DD-WRT software (build 49544) : 
Jan 22 06:50:36 VPNRouter daemon.err openvpn[1409]: 94.102.61.34:35806 TLS Error: TLS key negotiation failed to occur within 60 seconds (check your network connectivity)
Jan 22 06:50:36 VPNRouter daemon.err openvpn[1409]: 94.102.61.34:35806 TLS Error: TLS handshake failed

I did not change the configuration of openvpn log message format.

Reply:
May be, there's something that I did not understand in parser pattern usage. Let me know.

Reply:
`the machine name (or IP) is not used to trigger an event` it is your filtering at the top to make sure the machine name is `VPNRouter`? I triggered another comment.

Reply:
There is nothing wrong with your parser inparticular, it just the our syslog parser is not expecting the `daemon.err` as that not standard syslog logs

Reply:
VPNRouter is the name of my router. 
I do not mention it in my parser pattern. Or it is not my intention.
But you're right VPNRouter is present in the log file and assert file I use for the tests. May be I'm wrong but I understand that I need to provide some kind of syslog message in sample parser log file. it is real example.